rust/storage/src/admissioncontrolleds3.rs

```diff
@@ -92,7 +120,7 @@ impl AdmissionControlledS3Storage {
         storage: S3Storage,
         rate_limiter: Arc<RateLimitPolicy>,
         key: String,
-        options: GetOptions,
+        priority: Arc<AtomicUsize>,
```

Comment:
Should the caller just pass a priority and the fact that its arc and usize be internal to the module?

Reply:
oh this is internal nvm